src/klicky.ts

```diff
@@ -1,5 +1,21 @@
 export const Klicky = (dataSelector: string) => {
   document.addEventListener("click", function (evt) {
+    // Pointer event
+ document.addEventListener("click", function(event){
```

Comment:
You have an event listener happening right above this line, so you don't need to add it twice, just leave the console.log

Reply:
I removed the second event and left the console.log.